metriken-exposition/src/parquet.rs

```diff
@@ -73,23 +71,23 @@ impl ParquetOptions {
     /// Sets the compression level for the parquet file. The default is no
     /// compression. Set the compression level to a corresponding zstd level to
     /// enable compression.
-    pub fn compression(mut self, compression: ParquetCompression) -> Self {
+    pub fn with_compression(mut self, compression: ParquetCompression) -> Self {
```

Comment:
Why are we prefixing with `with_`? I'm not sure if this is idiomatic for builder pattern, which I think is what we have here?

Reply:
More for consistency with the rest of the parquet/arrow stuff, since they seem to use it. Not sure if other rust builders use that terminology and don't have a particular preference.

Reply:
Hmm, actually, looking at this, I'm not sure where I got the `with_` from -- the `ArrowWriter` builder seems to use `set_`. Shall we just use that since it is less ambiguous?

Reply:
I agree with keeping it bare (no with_ or set_) since it is chainable and we don't need to distinguish from methods which return the value.

Reply:
I don't love the fact that it isn't obvious whether it returns or sets the value, but I can live with it. Changing back to the previous names.